flink/src/main/java/org/apache/iceberg/flink/source/FlinkInputFormat.java

```diff
@@ -45,30 +43,25 @@
   private static final long serialVersionUID = 1L;
 
   private final TableLoader tableLoader;
-  private final Schema projectedSchema;
-  private final ScanOptions options;
-  private final List<Expression> filterExpressions;
+  private final SerializableConfiguration serializableConf;
   private final FileIO io;
   private final EncryptionManager encryption;
-  private final SerializableConfiguration serializableConf;
+  private final ScanContext context;
 
   private transient RowDataIterator iterator;
 
-  FlinkInputFormat(
-      TableLoader tableLoader, Schema projectedSchema, FileIO io, EncryptionManager encryption,
-      List<Expression> filterExpressions, ScanOptions options, SerializableConfiguration serializableConf) {
+  FlinkInputFormat(TableLoader tableLoader, SerializableConfiguration serializableConf, FileIO io,
+                   EncryptionManager encryption, ScanContext context) {
     this.tableLoader = tableLoader;
-    this.projectedSchema = projectedSchema;
+    this.serializableConf = serializableConf;
```

Comment:
Why pass the conf here? I don't see it used.

Reply:
It is used to open `TableLoader`.

Reply:
I see now. Thanks!